### PR TITLE
Adds integration tests using Cypress

### LINKED ISSUE #333: Cypress Test Suite
- Integration Test Suite set up to run against electron
- Should cover critical functionality of the builder

### High Level Test Areas
- Creation of apps
- Creation of a model
- View records for model
- Add a user
- Create some components
- Update their props
- Create a workflow

### PR DESCRIPTION
**Pull request by kevmodrome**
## Description
This adds cypress as well as a number of integration tests to the builder. Should close #333 

- [X] Creation of apps
- [x] Creation of a model
- [x] View records for model
- [x] Add a user
- [x] Create some components
- [x] Update their props
- [x] Create a workflow

## Screenshots
_If a UI facing feature, some screenshots of the new functionality._

### CODE REVIEW

**Comment by kevmodrome** [comment]
This is ready to be tested and merged if it works!

File: config.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+module.exports = () => ({
```
Comment (shogunpurple) [inline config.js:1]: We can remove this file, it's no longer relevant

**Review by shogunpurple — APPROVED** [review]
Nice work 🚀  just need to add these to the CI run 👍

**Comment by mjashanks** [comment]
Tests are running on windows, although we still have a few failures... still due to session not authenticated...

Will continue investigation...

image

**Pull request merged**